### ISSUE TITLE
msp430-gcc does not compile on yosemite

### ISSUE THREAD

**Issue by rasit** [sampsyo/homebrew-mspgcc#7]
$ brew install msp430-libc 

give the following error:

==> ../configure --target=msp430 --enable-languages=c,c++ --program-prefix='msp430-' --prefix=/usr/local/Cellar/msp430-g
==> make
checking for suffix of object files... configure: error: in `/private/tmp/msp430-gcc20150331-75858-1ubybp9/gcc-4.7.0/build/msp430/libgcc':
configure: error: cannot compute suffix of object files: cannot compile
See `config.log' for more details.
make[1]: *** [configure-target-libgcc] Error 1
make: *** [all] Error 2

My Mac is running up to date yosemite and xtools.

Any idea what might be causing this?

**Comment by sampsyo** [comment]
That certainly stumps me, without digging into the autoconf pieces myself. It looks like a bog-standard autotools artifact. Any input here would be welcome.

**Comment by rasit** [comment]
I tried on a pristine macbook with the same OS, but initially no brew. I installed brew and followed the instructions on this site, but got the very same result. I'm not sure what goes wrong.

**Comment by tsparber** [comment]
Hi,

I had the same error message. Looking at the config.log (-> /private/tmp/msp430-gcc20150803-13111-ykuof1/gcc-4.7.0/build/msp430/libgcc/config.log) the following output was generated:

```
configure:3625: checking for suffix of object files
configure:3647: /private/tmp/msp430-gcc20150803-13111-ykuof1/gcc-4.7.0/build/./gcc/xgcc -B/private/tmp/msp430-gcc20150803-13111-ykuof1/gcc-4.7.0/build/./gcc/ -B/usr/local/Cellar/msp430-gcc/4.7.0/msp430/bin/ -B/usr/local/Cellar/msp430-gcc/4.7.0/msp430/lib/ -isystem /usr/local/Cellar/msp430-gcc/4.7.0/msp430/include -isystem /usr/local/Cellar/msp430-gcc/4.7.0/msp430/sys-include    -c -g -Os -w -pipe -march=core2 -msse4.1  conftest.c >&5
xgcc: error: unrecognized command line option '-march=core2'
xgcc: error: unrecognized command line option '-msse4.1'
configure:3651: $? = 1
configure: failed program was:
| /* confdefs.h */
| #define PACKAGE_NAME "GNU C Runtime Library"
| #define PACKAGE_TARNAME "libgcc"
| #define PACKAGE_VERSION "1.0"
| #define PACKAGE_STRING "GNU C Runtime Library 1.0"
| #define PACKAGE_BUGREPORT ""
| #define PACKAGE_URL "http://www.gnu.org/software/libgcc/"
| /* end confdefs.h.  */
| 
| int
| main ()
| {
| 
|   ;
|   return 0;
| }
configure:3665: error: in `/private/tmp/msp430-gcc20150803-13111-ykuof1/gcc-4.7.0/build/msp430/libgcc':
configure:3668: error: cannot compute suffix of object files: cannot compile
See `config.log' for more details.
```

Editing the https://github.com/sampsyo/homebrew-mspgcc/blob/master/msp430-gcc.rb#L32 file to remove also the these environment options (-march=core2, -msse4.1) solved the problem for me (OSX yosemite 10.10.4 on a MacBook Pro Mid 2010, Core2Duo)

```
    ENV.remove_from_cflags(/ ?-march=\S*/)
    ENV.remove_from_cflags(/ ?-msse4(\.\d)?/)
```

**Comment by sampsyo** [comment]
Should be fixed in #11.
